rails_local.bash

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+source config/secrets/.source_env.bash
+
+thin start --ssl --ssl-key-file config/local_ssl/localhost.key --ssl-cert-file config/local_ssl/localhost.crt  --ssl-disable-verify
```

Comment:
Since we're using `puma` now, is this still needed?

Reply:
Nope, good catch, removed in 2657056.